apps/yapms/src/lib/stores/regions/Regions.ts

```diff
@@ -143,3 +146,46 @@ export const CandidateCountsMargins = derived(RegionsStore, ($RegionStore) => {
 	});
 	return candidates;
 });
+
+export const setPointerEvents = (): void => {
```

Comment:
Good point, I think just because it doesn't parse the node data. It was just reading the regions.

I think I'll make it so that more function use the region store instead of the raw nodes.